client-tools/Shell/ShellFeature.h

```diff
@@ -63,7 +81,9 @@ class ShellFeature final : public ArangoshFeature {
   std::vector<std::string> _positionals;
   std::string _unitTestFilter;
   std::vector<std::string> _scriptParameters;
+  std::unique_ptr<TelemetricsHandler> _telemetricsHandler;
   bool _runMain{false};
+  bool _sendToEndpoint{false};
```

Comment:
Should this only be present in maintainer mode?
The member should also be renamed to something more telling. `sendToEndpoint` is nice, but send **what**? Tribbles?
```suggestion
  bool _sendToEndpoint{false};
```